lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -27,6 +27,21 @@ using namespace mlir::torch::onnx_c;
 // to be more normal and a direct translation vs a special case. This
 // results in a lot of ONNX test cases that all reduce to the exact same
 // thing here, so we simplify.
+
+// utilities
+//  Templatized function to get an item op of a type
+namespace {
+template <typename T>
+Value getItemOp(OpBinder binder, ConversionPatternRewriter &rewriter,
+                Value &ofItem);
+template <typename T>
+Value getItemOp(OpBinder binder, ConversionPatternRewriter &rewriter,
+                Value &ofItem) {
+  return rewriter.create<Torch::AtenItemOp>(binder.getLoc(),
+                                            rewriter.getType<T>(), ofItem);
```

Comment:
I think there's no need to declare this utility here since you're also defining it here.

Reply:
That is right, was thinking of having a common place for other CPP files to share initially. Declaration removed. I see that createTorchTransposeOp and onnxDtypeIntToTorchDtypeInt are static fucntions in /torch-mlir/lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp but they could be useful for other Domain*cpp files too. Maybe we should have DefaultDomainCommon.h and .cpp and put common utility functions there?

Reply:
Yeah, that's a good idea. We can do that in a follow-up PR.